### ISSUE TITLE
Convert the UI to use keyboard input

### ISSUE THREAD

**Issue by zjcers** [EECS581Team15/WifiLMR#32]
The UI MVP uses mouse input for button presses. As we will not have a mouse or any other sort of pointing device, we need to convert it use keyboard navigation. Specifically, there will be five keys: up, down, right, left, and enter. This dictates that there will be a `current` selected control. This control should be highlighted in some way and pressing enter should activate it.

**Comment by zjcers** [comment]
tkinter already supports Tab and Shift+Tab for navigation and Space for activating controls. We can map the left, right, and select keys to those event types and this should mostly work. One thing I noticed is that widgets that are from the previous screen are still accessible. This is another issue that should be addressed anyway.

**Comment by zjcers** [comment]
Stealing this.
